mne/morph.py

```diff
@@ -567,6 +567,9 @@ def read_source_morph(fname):
         morph = vals['sdr_morph']
         vals['sdr_morph'] = DiffeomorphicMap(None, [])
         vals['sdr_morph'].__dict__ = morph
+    # Backward compat with when it used to be a list
+    if isinstance(vals['vertices_to'], np.ndarray):
+        vals['vertices_to'] = [vals['vertices_to']]
```

Comment:
do we still need this?

Reply:
Yes anyone with a SourceMorph saved to disk will need this shim